tests/test_validation.py

```diff
@@ -483,13 +490,13 @@ def test_missing_fieldnames(self):
         vector = None
 
         error_msg = validation.check_vector(
-            filepath, required_fields=['col_a', 'COL_B', 'col_c'])
-        self.assertTrue('Fields are missing' in error_msg)
-        self.assertTrue('col_c'.upper() in error_msg)
+            filepath, field_patterns=['col_a', 'COL_B', 'col_c'])
+        self.assertTrue(
+            'matched 0 headers, expected at least one' in error_msg)
```

Comment:
Same comment as many others, but here's the more general case. Can we rephrase the message to be more helpful to users by reminding them of the correct field names?

Reply:
see https://github.com/natcap/invest/pull/526#discussion_r652766326